blog/models.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.conf import settings
+
+# Create your models here.
+class User(AbstractUser):
+    pass
+
+
+class Post(models.Model):
+    owner = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, related_name='posts')
```

Comment:
It's a good practice to add help_text to your model fields. This will provide a text description for each field, which can be useful in the Django admin site and other forms. For example: `owner = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, related_name='posts', help_text='User who created the post')`